R/test-files.R

```diff
@@ -98,6 +94,22 @@ test_dir <- function(path,
   reporter <- find_reporter(reporter)
   parallel <- want_parallel && reporter$capabilities$parallel_support
 
+  if (parallel) {
+    if (!is_missing(wrap)) {
```

Comment:
Given that you're the only person using this argument, I think you can just keep the warning the same for both serial and parallel.

Reply:
Ok, I did this.  If for whatever reason someone ends up using `wrap=FALSE` in parallel mode they might be confused, but given they will get a deprecation warning hopefully they'll figure it out, and more likely such a person won't exist anyway.